synfig-studio/src/gui/trees/layertree.cpp

```diff
@@ -242,7 +242,7 @@ LayerTree::~LayerTree()
 		synfig::info("LayerTree::~LayerTree(): Deleted");
 }
 
-Gtk::Widget*
+void
 LayerTree::create_layer_tree()
```

Comment:
Comment at line 161 will never be uncommented and should be removed https://github.com/synfig/synfig/pull/888/commits/c0d21e73ad13e3dc5ba4e897274e74792945b594#diff-fd6c1c2f0e4b0e2fda5ac15a95dee389L171